cpu/imxrt117x-cm4/imxrt117x-cm4.c

```diff
@@ -233,9 +233,9 @@ static int runCore(void)
 		return -EIO;
 
 	/* Release the Kraken */
-	*(m4_common.scr + scr_smsr) |= 0xf << 10;
+	*(m4_common.src + src_srmr) |= 0xf << 10;
```

Comment:
If the M7 core is the primary `LR` role and the M4 is the secondary `NLR` role, this needs to be changed.
Setting bits 12-13 (M7REQ_RESET_MODE) to "do not reset anything" prevents the NVIC from resetting the M7 core (it will hang, due to the "do not reset anything" bits set), the reboot command will not work (it will freeze the core until valid reset from other source comes, you will be able to do a reset only via other enabled sources TEMP, OVERVOLT, CSU, WDOG, JTAG because these bits are 00 = "reset system - both M4 and M7").

So, set 10-11 (M4REQ_RESET_MODE) just to prevent M4 core from resetting both cores (suspend M4 during reset), and leave 12-13 bits as 0 to allow M7 core to reset system (as M7 is the Legally Relevant core)
```suggestion
	*(m4_common.src + src_srmr) |= 0x3u << 10;
```
and/or it may be required to set M4_LOCKUP_RESEST_MODE (bits 5-6) not to reset the system (both cores) during M4 lockup. The M7 core (as LR core) can reset the M4 core (NLR core) as needed:
```suggestion
	*(m4_common.src + src_srmr) |= (0x3u << 10) | (0x3u << 6);
```